x/multisig/model.go

```diff
@@ -69,3 +69,10 @@ func (b ContractBucket) Save(db weave.KVStore, obj orm.Object) error {
 	}
 	return b.Bucket.Save(db, obj)
 }
+
+// Build assigns an ID to given contract instance and returns it as an orm
+// Object. It does not persist the escrow in the store.
+func (b ContractBucket) Build(db weave.KVStore, c *Contract) orm.Object {
```

Comment:
👍 
q: Was `Build` supposed to save the entity?

Reply:
Yes, this is almost like `Create` but without saving. It's used in [escrow](https://github.com/iov-one/weave/blob/master/x/escrow/model.go#L163) as well.